enderio-base/src/main/java/com/enderio/base/client/gui/screen/IEnderScreen.java

```diff
@@ -16,10 +16,18 @@ default Screen getScreen() {
     }
 
     default void renderIcon(PoseStack pPoseStack, Vector2i pos, IIcon icon) {
```

Comment:
Why do we have this if it just passes to a static method?

Reply:
good question. I'll remove it